audit/src/main/java/org/wildfly/security/audit/SyslogAuditEndpoint.java

```diff
@@ -116,6 +143,8 @@ public static Builder builder() {
         private boolean tcp = true;
         private String hostName;
         private SocketFactory socketFactory = null;
+        private SyslogHandler.SyslogType format = SyslogHandler.SyslogType.RFC5424;
+        private int maxReconnectAttempts = 0;
```

Comment:
The comments on the analysis document mention that the default should be -1. However, the description in the analysis document itself wasn't updated.